Client/Frontend/Browser/Playlist/VideoPlayer/PlaylistParticleEmitter.swift

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import UIKit
+
+class PlaylistParticleEmitter: UIView {
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        if let layer = self.layer as? CAEmitterLayer {
+            let cells = generateEmitterCells()
+            cells.base.emitterCells = [cells.trailing, cells.particle]
+            layer.emitterCells = [cells.base]
+        }
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override class var layerClass: AnyClass {
+        return CAEmitterLayer.self
```

Comment:
`return` can be ommited

Reply:
`GradientView` (and a lot of our older code) was written before Xcode added single-line returns, we'll eventually get to clean those up, but at least we can make sure any new code we write uses it